packages/theme/src/components/post/CreatePost.vue

```diff
@@ -11,14 +11,19 @@
       :disabled="createPostPermissionDisabled"
       @keyup-enter="submitPost"
       @hide-error="hideTitleError"
+      @input="handleTitleInput"
     />
+    <HelperText v-if="title.value.length >= 85 && description.length <= 15">
+      After the title limit, you’ll automatically jump to description.
```

Comment:
@TaranaKhanna what other alternative text do you suggest we can write. The text should be simple, and descriptive.